packages/celer-web-app/src/ui/components/DocLineComponent/DocLineTextComponent.tsx

```diff
@@ -196,6 +272,31 @@ export const DocLineTextWithIconComponent: React.FC<DocLineTextWithIconProps> =
 			textStyleName = styles.instructionMoldugaColor;
 			break;
 	}
+
+	if (collapseNotes && settings.setting.collapseNotes) {
+		return (
+			<div>
+				<div className={clsx(styles.lineContainer, altLineColor && styles.lineContainerAlt)}>
+					<LineNumberWithIcon docLine={docLine} />
+					<Counter docLine={docLine} />
+					<StepNumberWithIcon docLine={docLine}/>
+					<div className={clsx(styles.instructionNotesCollapsed, styles.instructionWithIconNotesCollapsed, textStyleName)}>
+						<div className={styles.icon}>
+							<img width={"100%"} height={"auto"} src={Icons[icon]} alt={icon}/>
+						</div>
+						<div className={styles.iconSideText}>
+							<TypedStringComponent content={text} variables={variables} isNotes={false}/>
+							<div className={clsx(styles.commentFont, styles.commentColor)}>
+								{comment && <TypedStringComponent content={comment} variables={variables} isNotes={false}/>}{"\u200b"}
+							</div>
+						</div>
+					</div>
+					<NotesCollapsed docLine={docLine} altNotesColor={altNotesColor} bannerOpen={notesBannerOpen} toggleBanner={setNotesBannerOpen}/>
+				</div>
+				{notesBannerOpen && <NoteBannerComponent docLine={docLine} altNotesColor={altNotesColor}/>}
```

Comment:
Since the notes is part of this line, the if the user uses the keyboard to navigate the lines, it would skip both the text and the notes with one key, right? (This should be the expected behavior but idk if you tested it)

Reply:
Correct, it works as intended.